src/Store/SessionStore.php

```diff
@@ -145,6 +151,8 @@ public static function clear(HTTPRequest $request)
     protected function resetMethod()
     {
         $this->setMethod(null)->setState([]);
```

Comment:
Thoughts on renaming `setState()` to `setMethodState()`, and letting `setMethod()` to clear its method state when set to falsy?